sources/Inputs/include/Configuration.h

```diff
@@ -133,6 +170,8 @@ class Configuration {
   }
 
  private:
+  using seconds = std::chrono::seconds;  ///< Alias for seconds
```

Comment:
DFL types are in CamelCase

Reply:
Type has not been replaced in getStartTime and getStopTime => this requires to put the type in public